polyfill/lib/zoneddatetime.mjs

```diff
@@ -453,8 +467,9 @@ function zonedDateTimeToString(zdt, precision, options = undefined) {
   minute = ES.ISODateTimePartString(minute);
   const seconds = ES.FormatSecondsStringPart(second, millisecond, microsecond, nanosecond, precision);
   const tz = GetSlot(zdt, TIME_ZONE);
-  const offset = ES.GetOffsetStringFor(tz, GetSlot(zdt, INSTANT));
-  const zone = ES.TimeZoneToString(tz);
-  const calendar = ES.FormatCalendarAnnotation(GetSlot(zdt, CALENDAR));
-  return `${year}-${month}-${day}T${hour}:${minute}${seconds}${offset}[${zone}]${calendar}`;
+  let result = `${year}-${month}-${day}T${hour}:${minute}${seconds}`;
```

Comment:
I noticed that the `__repr__` property isn't using the same format as here. Is this intentional?

```js
ZonedDateTime {_repr_: "Temporal.ZonedDateTime <2020-11-01T01:30:00.0-07:00[America/Los_Angeles]>"}
```

Note the `.0` after the seconds, which looks odd.

Reply:
I added a commit fixing this.